task/prefetch-dependencies/0.1/prefetch-dependencies.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "image-build, appstudio, hacbs"
+  name: prefetch-dependencies
+spec:
+  description: |-
+    Task that uses Cachi2 to preftech build dependencies.
+  params:
+  - description: Package type
+    name: package-type
+  - description: Package path
+    name: package-path
+  - description: Flags
+    name: flags
+  steps:
+  - image: quay.io/containerbuildsystem/cachi2:e2925aa
```

Comment:
How will the image be updated? (Pardon my HACBS ignorance)

Or is this something to be revisited after we get Cachi2 container builds working properly? (Currently this is the only existing image https://quay.io/repository/containerbuildsystem/cachi2?tab=tags)

Reply:
As of now, the images on Tasks are being updated manually. I imagine this will also be the case for the prefetch step. In any case, I created a ticket so we can design and document a process to do it within Cloud Build.